web/src/pages/Upload.jsx

```diff
@@ -0,0 +1,143 @@
+import { UploadOutlined, CloudUploadOutlined } from "@ant-design/icons";
+import { Button, message, Upload, Card, Typography, Divider } from "antd";
+import { useState } from "react";
+
+const { Title, Text } = Typography;
+
+export const FileUploading = () => {
```

Comment:
Could we rename this component to `FileUploadPage` to maintain consistency with our other page components?